types/sdk.js

```diff
@@ -60,7 +60,7 @@ export type AuthMiddlewareOptions = {
   scopes: Array<string>;
 }
 export type HttpMiddlewareOptions = {
-  host?: string;
+  host: string;
```

Comment:
Sure. The only thing is that this type can not really be "used" for documentation, since it says it's a required field.